static/src/javascripts/projects/admin/bootstraps/commercial-browser-performance.js

```diff
@@ -0,0 +1,104 @@
+// @flow
+/* global $, Epoch*/
+import config from 'lib/config';
+import fetchJson from 'lib/fetch-json';
+import template from 'lodash/utilities/template';
+import countBy from 'lodash/collections/countBy';
+import find from 'lodash/collections/find';
+
+// Globals that aren't imported
+declare var $: any;
+declare var Epoch: any;
+
+let chart;
+const FETCH_INTERVAL = 1000; // The frequency that we poll for report data.
+const FETCH_DELAY = 10; // The delay which we wait before we ask for a time-based datapoint, eg. 10 seconds before the present moment.
+const reportTemplateUrl = '/commercial-reports/<%=isoDate%>';
+
+// Store the 1000 most recently fetched datapoints.
+const commercialStartTimes = [];
+
+const updateAverageStartTime = startTimes => {
+    // Push the new start times into the stored array to find an average.
+    Array.prototype.push.apply(commercialStartTimes, startTimes);
+    // Limit the size of the array to 1000.
+    if (commercialStartTimes.length > 1000) {
+        commercialStartTimes.splice(0, commercialStartTimes.length - 1000);
+    }
+
+    if (!commercialStartTimes.length) {
+        return;
+    }
+
+    const sum = commercialStartTimes.reduce((accum, num) => {
+        // Disregard silly numbers, investigating why Date times are appearing in the array.
+        if (num < 20000) {
+            return accum + num;
+        }
+        return accum;
+    });
+
+    const averageStartTime = (sum / commercialStartTimes.length).toFixed(2);
```

Comment:
This has changed from `.toFixed([2])` to `.toFixed(2)` because according to flow, `toFixed` takes a `number`.
@rich-nguyen Was there a reason this was an array?